docs/help/FAQ/10_Password_Complexity.rst

```diff
@@ -17,17 +17,32 @@ The default complexity is enforced in both :term:`PAM` and :term:`LDAP`.
 Complexity Rules
 ----------------
 
-  * 14 Characters or greater
+  * 15 Characters or greater
   * 1 Upper case letter
   * 1 Lower case letter
   * 1 Number
   * 1 Special character
   * No more than 2 repetitions of the same character
+  
+    * OK: ``aab``
+    * BAD: ``aaa``
+
+  * No more than 3 repetitions of a character from the same character class
+  
+    * OK: ``abc``
+    * BAD: ``abcd``
+
   * No more than 4 characters in a monotonic character sequence
+  
+    * OK: ``1b2c3d``
+    * BAD: ``aBcDe``
```

Comment:
I just tested against the pam module's defaults, using the acceptance tests:
  * The example ``aBcDe`` fails (as expected) against el6/`cracklib`, but **passes** against el7/`pw_quailty`.
  * A descending sequence of >4 chars like ``FeDcB`` fails against el6/`cracklib`, but **passes** against el7/`pw_quailty`.

Reply:
:man_facepalming: I think I'm going to note that behavior may differ slightly between `cracklib` and `pw_quality`. The man page says that it still works the same way so it's probably just a bug.